helm-chart/templates/service-monitor.yaml

```diff
@@ -0,0 +1,23 @@
+{{ if .Values.serviceMonitor.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ include "promscale.fullname" . }}
+  namespace: {{ template "promscale.namespace" . }}
+  labels:
+    app: {{ template "promscale.fullname" . }}
+    chart: {{ template "promscale.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  endpoints:
+  - interval: 30s
```

Comment:
Any specific reason for configuring the default to `30s` not letting it configure from `values.yaml`?

Reply:
This is the default. If we get more users of this, we can improve. Either way, this should be opinionated and related to alerts that we are working on.